src/pcp/pidstat/pcp-pidstat.py

```diff
@@ -352,10 +352,12 @@ def __init__(self, cpu_usage, process_filter, delta_time, printer, pidstat_optio
         self.pidstat_options = pidstat_options
 
     def print_report(self, timestamp, ncpu):
+        indentation = "        " if len(timestamp)<9 else (len(timestamp)-7)*" "
```

Comment:
ISTM we do the same calculation for indentations several times, could we refactor this into a single calculation where we set the timestamp into two variables, header_indentation and value_indentation ?  Then perhaps pass it to each print_report as needed?